### PR TITLE
#171123089 Coveralls Fixing 

### PR DESCRIPTION
**Pull request by Niyonsengaeric**
#### What does this PR do?
This PR fixes  Coveralls badge error.
#### Description of Task to be completed?
- Running tests
- Checking coverage badge
#### How should this be manually tested?
- run ` git clone https://github.com/andela/knights-bn-backend.git ` in your terminal 
- run `cd knights-bn-backend`
- run `npm i` to install the dependencies 
- run `npm run coverage` to track your code coverage
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#171123089](https://www.pivotaltracker.com/story/show/171123089)

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

**Review by EugeneMunya — COMMENTED** [review]
Try to squash your commits

**Comment by Moise1** [comment]
LGTM

**Review by Moise1 — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,15 @@
-import dotenv from 'dotenv';
 import app from './app';
 
-dotenv.config();
```
Comment (MuhanguziDavid) [inline src/index.js:4]: Is there a reason for removing this line? We may need this package.

**Comment by MuhanguziDavid** [comment]
Your PR description does not seem to be related to this task at all.

**Comment by ceelogre** [comment]
Please review each other's work properly and thoroughly before leaving any comment.

**Review by Kagorora — APPROVED** [review]

**Review by EugeneMunya — APPROVED** [review]

**Review by Moise1 — APPROVED** [review]

File: src/index.js [hunk 2]
```diff
@@ -2,5 +2,16 @@ import dotenv from 'dotenv';
 import app from './app';
 
 dotenv.config();
+app.use((req, res) => {
```
Comment (T2Wil) [inline src/index.js]: @Niyonsengaeric ,
I think this should be moved into src/app.js where we configured and exported express app

**Comment by MuhanguziDavid** [comment]
@Niyonsengaeric , I get the coverage report twice on running `npm run coverage`. There may be a repeated command in your coverage script.

Screenshot 2020-02-06 at 18 53 57

**Review by T2Wil — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (Niyonsengaeric) [reply src/index.js]: Moved 👍

**Review by T2Wil — APPROVED** [review]

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Pull request merged**